front/src/applications/operationalStudies/types.ts

```diff
@@ -95,6 +96,7 @@ export type ManageTrainSchedulePathProperties = {
   allWaypoints: SuggestedOP[];
   length: number;
   trackSectionRanges: NonNullable<PathfindingResultSuccess['track_section_ranges']>;
+  incompatibleConstraints?: Partial<IncompatibleConstraints>;
```

Comment:
Seems like the endpoint always returns all 3 constraints but they are empty arrays if constraint is not present

```suggestion
  incompatibleConstraints?: IncompatibleConstraints;
```